database/index.js

```diff
@@ -1,11 +1,13 @@
 const mongoose = require('mongoose');
 
-mongoose.connect('mongodb://localhost/menuService', { useNewUrlParser: true });
+const PORT = 27017;
+
+mongoose.connect(`mongodb://localhost:${PORT}/menuService`, { useNewUrlParser: true });
```

Comment:
Blocked: Could you explain why you changed 'mongodb://localhost/menuService' to `mongodb://localhost:${PORT}/menuService`?

Reply:
This is something we may or may not all need to think about when we make our proxy servers. 

If you don't assign the DB to a port, it will use your computers default. Jeremy and I both looked our default up and they both happened to be 27017. This makes me think that we will not have problems once we connect our proxy server to multiple DB's. If we happen to be using different ports for 2 different mongo DB instances, I'm not sure if that will cause a problem or not. For now, I just chose to be explicit and assign my db instance to the default port.